pyproject.toml

```
@@ -250,12 +249,7 @@ cover.help = "Run the test suite with coverage"
 cover.ref = "test --cov-report term-missing --cov-report=xml:coverage.xml --cov=commitizen"
 
 all.help = "Run all tasks"
-all.sequence = [
-    "format",
-    "lint",
-    "cover",
-    "check-commit"
-]
+all.sequence = ["format", "lint", "cover", "check-commit"]
```

I tend to avoid 1-liner list as in the case of addition, change or removal, the entire line is changed, while the single item by line allows to precisely see what has been added, changed or removed.
To avoid being reformatted, add the missing trailing comma:
```suggestion
all.sequence = [
    "format",
    "lint",
    "cover",
    "check-commit",
]
```

---

opps, didn't notice my editor format it for me...